examples/maildev/index.js

```diff
@@ -12,11 +12,47 @@ module.exports = async function (fastify, options) {
   fastify.register(fastifyMail, { pov: { engine: { nunjucks } }, transporter })
 
   fastify.get('/sendmail', async (req, reply) => {
-    const recipients = ['test@example.com']
-    const templates = './templates'
-    const context = { name: 'Test Name', sender: 'sender@example.com' }
+    const { name } = req.query
```

Comment:
let's consider simplifying for our demonstration purposes, keep things simple for the evaluator of the plugin

Reply:
done